### DISCUSSION TITLE
Doubt regarding soundcloud

### DISCUSSION THREAD

**Discussion by St3am854** [Shabinder/SpotiFlyer#863] [Q&A]
How do I download songs from soundcloud.
The links keep getting rejected....

**Comment by Shabinder** [comment]
Share which link you are trying ?

**Reply by St3am854** [reply]
Listen to Bad Intentions (feat. Migos) by NiykeeHeaton on #SoundCloud
https://soundcloud.app.goo.gl/QEL6E

**Comment by yavnaz** [comment]
Same here; this is one of MANY that's not downloading. ERROR MSG: Polymorphic Serializer was not found for class discriminator 

Power Up the 90s club mix on #SoundCloud
https://soundcloud.app.goo.gl/MA2eG
